client/python/lib/ovmsclient/tfs_compat/serving_client.py

```diff
@@ -0,0 +1,230 @@
+#
+# Copyright (c) 2021 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+class ServingClient:
+    
+    def __init__(self, address, grpc_port, http_port=None):
+        raise NotImplementedError
+
+    def send_predict_request(self, request):
```

Comment:
why don't we call it simply "Predict"

Reply:
I chose that name to be as descriptive as possible about what a method does. 
I didn't use "predict" as it might be used by an API on the higher level.